masterlist.yaml

```
@@ -13180,7 +13180,7 @@ plugins:
           - '[Wintersun - More Patches](https://www.nexusmods.com/skyrimspecialedition/mods/24228/)'
         condition: 'active("Wintersun - Faiths of Skyrim.esp") and not active("Wintersun - BCS Patch.esp") and not active("Bashed Patch.*\.esp")'
       - <<: *patch3rdParty_KPH_TCIY
-        condition: 'active("TheChoiceIsYours.esp") and not active("TCIY BCS Patch.esp") and not active("Bashed Patch.*\.esp")'
+        condition: 'active("TheChoiceIsYours.esp") and not active("TCIY( |_)BCS Patch.esp") and not active("Bashed Patch.*\.esp")'
```

@FWDekker Since the expression is a regex now, the period should be escaped:

```suggestion
        condition: 'active("TheChoiceIsYours.esp") and not active("TCIY( |_)BCS Patch\.esp") and not active("Bashed Patch.*\.esp")'
```